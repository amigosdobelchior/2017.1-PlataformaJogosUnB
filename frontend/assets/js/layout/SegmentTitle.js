import React from "react";
import PropTypes from 'prop-types';
import { Segment } from 'semantic-ui-react';

export default class SegmentTitle extends React.Component {
	render() {
		return (
			<Segment padded inverted color="grey">
				<h1>{this.props.title}</h1>
			</Segment>
		);
	}
}

SegmentTitle.propTypes = {
    title: PropTypes.array.isRequired,
}