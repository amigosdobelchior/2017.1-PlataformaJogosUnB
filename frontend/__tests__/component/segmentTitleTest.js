import React from 'react';
import SegmentTitle from '../../assets/js/layout/SegmentTitle';
import {Dropdown} from "semantic-ui-react";
import { Segment } from 'semantic-ui-react';
import renderer from 'react-test-renderer';
var expect = required('expect');

test('Test render SegmentTitle', () => {
  const component = renderer.create(
      <Segment padded inverted color="grey">
          <h1>"Titulo"</h1>
      </Segment>
	);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
