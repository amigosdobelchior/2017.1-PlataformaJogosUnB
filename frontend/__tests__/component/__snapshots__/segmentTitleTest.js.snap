// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render SegmentTitle 1`] = `
<div
  className="ui grey inverted padded segment"
>
  <h1>
    Titulo
  </h1>
</div>
`;
